suricata.yaml.in

```diff
@@ -860,6 +860,8 @@ pcap-file:
   #  checksum off-loading is used. (default)
   # Warning: 'checksum-validation' must be set to yes to have checksum tested
   checksum-checks: auto
+  # Read buffer size set using setvbuf. Max value is 64 MiB. Linux only.
+  #buffer-size: 128 KiB
```

Comment:
This sounds confusing/contradicting compared to the [doc/userguide/partials/options.rst](https://github.com/OISF/suricata/pull/11619/files#diff-b977343c14bb5d703a435c8fc1caa374b2414ca32ba0656f96ba2b7adb5b92fa) mentioned above. Is "setvbuf" only on linux? or is the max value different on linux? why is the default 128 if the max is 64?

Reply:
`setvbuf` is in standard C, but I think its only reliable for pcap under Linux, or so in our testing?

Also in this case we have a max, just because it makes sense to have one. Even tho if its not an optimal value, 128k is a good value in the optimal range.

The config file is not a place to discuss these points, but we also don't have this configuration option in the userguide anywhere, where it would be a place to discuss these.

Reply:
Added a ticket, so we won't forget about this https://redmine.openinfosecfoundation.org/issues/7217